gui/include/gui/compass.h

```diff
@@ -74,11 +85,17 @@ class ocpnCompass {
   int m_yoffset;
   float m_scale;
 
+  /**
```

Comment:
It seems the right thing to do would be to change the units of `m_rect` from physical pixels to logical pixels.

Reply:
We are kind of stuck here as long as we support building against wx 3.0. Basically everything should be sized by device independent pixels and using `FromDIP` on wx3.2 to scale, but we can't do that until we drop support for wx3.0 build. As much as we would like this, there are still major Linux distributions that ship with 3.0, so it is not a no-brainer, but sure it is coming.